pkg/querier/queryrange/codec.go

```diff
@@ -1314,7 +1314,7 @@ func toProtoScalar(v loghttp.Scalar) []queryrangebase.SampleStream {
 	res = append(res, queryrangebase.SampleStream{
 		Samples: []logproto.LegacySample{{
 			Value:       float64(v.Value),
-			TimestampMs: v.Timestamp.UnixNano() / 1e6,
+			TimestampMs: v.Timestamp.Unix(),
```

Comment:
Is this returning milliseconds? I'll take a look in a bit

Reply:
i changed the PR to return seconds from `evalLiteral`. ptal.

Reply:
So, `v.Timestamp` is using the Prometheus `model.Time` which is in milliseconds. So this should be `TimestampMs: int64(v.Timestamp)`.